module/cpf.sh

```diff
@@ -325,46 +328,46 @@ function cpf() {
             fi
         done < <(echo "${fmtstr}"|grep -oE '%{[^}]+}')
 
-        #echo "XXX fmt >>> ${fmtstr} <<<" > /dev/stderr
-        #echo "XXX arg ${#args[@]}: ${args[@]}" > /dev/stderr
-        #echo "XXX sym ${#prefix[@]}: ${prefix[@]}" > /dev/stderr
+        #echo "XXX fmt >>> ${fmtstr} <<<" >${STDERR}
+        #echo "XXX arg ${#args[@]}: ${args[@]}" >${STDERR}
+        #echo "XXX sym ${#prefix[@]}: ${prefix[@]}" >${STDERR}
         local -i substitutions=$(echo ${fmtstr}|sed -e 's/%{\([^}]*\)}/\1/g'|tr -c -d '%'|wc -c)
-        #echo "XXX [ ${substitutions} == ${#args[@]} ]" > /dev/stderr
+        #echo "XXX [ ${substitutions} == ${#args[@]} ]" >${STDERR}
         if [ ${substitutions} -eq ${#args[@]} ]; then
             if ! echo "${fmtstr}"|grep -qE '%{'; then
                 local -i i
                 for ((i=0; i<${#args[@]}; i++)); do
-                    #echo "XXX pre-change arg $i  >>> ${args[$i]}" > /dev/stderr
+                    #echo "XXX pre-change arg $i  >>> ${args[$i]}" >${STDERR}
                     args[${i}]="${prefix[$i]}${args[${i}]}"
-                    #echo "XXX post-chance arg $i >>> ${args[$i]}" > /dev/stderr
+                    #echo "XXX post-chance arg $i >>> ${args[$i]}" >${STDERR}
                 done
                 #. XXX
-                #echo format is "${fmtstr}" 2>/dev/stderr
-                #echo args are "${args[@]}" 2>/dev/stderr
+                #echo format is "${fmtstr}" 2>${STDERR}
+                #echo args are "${args[@]}" 2>${STDERR}
                 printf "${fmtstr}" "${args[@]}"
             else
-                echo "CPF Failure - still have %{ in the fmtstr!: ${fmtstr}" > /dev/stderr
+                echo "CPF Failure - still have %{ in the fmtstr!: ${fmtstr}" >${STDERR}
                 exit 99
             fi
         else
-            echo "CPF Failure: mismatched arguments for given format string ( ${substitutions} in fmtstr, ${#args[@]} arguments supplied )" > /dev/stderr
-            echo "Formatstr: \`${fmtstr}'" > /dev/stderr
-            echo "Arguments:" > /dev/stderr
-            printf " * \`%s'\n" "${args[@]}" > /dev/stderr
+            echo "CPF Failure: mismatched arguments for given format string ( ${substitutions} in fmtstr, ${#args[@]} arguments supplied )" >${STDERR}
+            echo "Formatstr: \`${fmtstr}'" >${STDERR}
+            echo "Arguments:" >${STDERR}
+            printf " * \`%s'\n" "${args[@]}" >${STDERR}
             #exit 99
         fi
     else
         echo
     fi
 
-    #echo "XXX FUNC RETN" > /dev/stderr
+    #echo "XXX FUNC RETN" >${STDERR}
     [ ${g_DEBUG} -eq 0 ] || set -x
 }
 #. }=-
 #. theme -={
 function theme() {
     if [ $# -gt 0 ]; then
-        local dvc=/dev/stdout
+        local dvc=${STDOUT}
```

Comment:
Could do that too, just make sure to use the `${FD_STDOUT?}` so bash errors if you misspelled the variable name or it wasn't defined.